packages/notifications-library/src/notifications_library/_email_render.py

```diff
@@ -37,11 +37,12 @@ def render_email_parts(
     event_name: str,
     *,
     user: UserData,
+    sharer: SharerData,
```

Comment:
this function is general. You cannot enforce a sharer on all emails. It should be part of `other_data` instead